code_schemes/kalobeyei_s01_close_out.json

```diff
@@ -0,0 +1,300 @@
+{
+    "SchemeID": "Scheme-8f22bcba",
+    "Name": "S01 Close Out",
+    "Version": "0.0.0.1",
+    "Codes": [
+        {
+            "CodeID": "code-NOC-4eb70633",
+            "ControlCode": "NOC",
+            "StringValue": "NOC",
+            "DisplayText": "Noise (Other Channel)",
+            "VisibleInCoda": true,
+            "NumericValue": -60,
+            "CodeType": "Control"
+        },
+        {
+            "CodeID": "code-E-720cdb66",
+            "CodeType": "Meta",
+            "MetaCode": "escalate",
+            "DisplayText": "meta: ESCALATE",
+            "NumericValue": -100080,
+            "StringValue": "escalate",
+            "VisibleInCoda": true,
+            "Shortcut": "e"
+        },
+        {
+            "CodeID": "code-Q-a5d3700d",
+            "CodeType": "Meta",
+            "MetaCode": "question",
+            "DisplayText": "meta: question",
+            "NumericValue": -100010,
+            "StringValue": "question",
+            "VisibleInCoda": true,
+            "Shortcut": "q"
+        },
+        {
+            "CodeID": "code-S-461bfcdf",
+            "CodeType": "Meta",
+            "MetaCode": "statement",
+            "DisplayText": "meta: statement",
+            "NumericValue": -100090,
+            "StringValue": "statement",
+            "VisibleInCoda": true,
+            "Shortcut": "s"
+        },
+        {
```

Comment:
use the code scheme structure for leap and no the one in oxfam

Reply:
updated thanks.